config/samples/mongodb.com_v1_mongodbcommunity_disabled_process_cr.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: mongodbcommunity.mongodb.com/v1
+kind: MongoDBCommunity
+metadata:
+  name: example-mongodb
+spec:
+  members: 3
+  automationConfigOverride:
```

Comment:
why do we have `automationConfigOverride.automationConfig`, a 2-level attribute? are you planning on adding any kind of siblings to `automationConfig` attribute in the future?

Reply:
this was because of what I thought was a limitation, but I think actually we can keep it at 1 level depth. I will experiment with it.